### PR TITLE
feat: publish zap receipts for non-game note zaps

### PR DESCRIPTION
**Pull request by bonomat**

### CODE REVIEW

File: src/routes.rs [hunk 1]
```diff
@@ -299,6 +299,23 @@ pub(crate) async fn get_invoice_for_zap_impl(
...
+        invoice,
+        request: zap_request.clone(),
+        multiplier_note_id: String::new(),
+        nonce_commitment_note_id: EventId::all_zeros(),
+        bet_state: BetState::ZapInvoiceRequested,
+        index: 0,
+        bet_timestamp: OffsetDateTime::now_utc(),
+    };
+
+    // At this stage, this `Zap` indicates the roller's _intention_ to bet. They have until the zap
```
Comment (holzeis) [inline src/routes.rs:315]: 🔧 this comment does not belong here.

File: src/subscriber.rs [hunk 2]
```diff
@@ -155,65 +165,73 @@ async fn handle_paid_invoice(
...
+        options
+            .wait_for_send(true)
+            .send_timeout(Some(Duration::from_secs(20))),
+    );
+    let relays = og_client.relays().await;
+    let relays = relays.keys();
+    client.add_relays(relays).await?;
+    client.add_relays(utils::get_relays(&zap.request)?).await?;
+    client.connect().await;
+    client.set_zapper(og_client.zapper().await?).await;
```
Comment (holzeis) [inline src/subscriber.rs:235]: 🤔 you shouldn't need a zapper if we just want to publish a zap receipt.

**Review by holzeis — APPROVED** [review]
LGTM 👍

File: src/subscriber.rs (hunk 2 above)
Comment (bonomat) [reply src/subscriber.rs:235]: The client is also used to zap the winners on the other branch

**Pull request merged**